identity/src/lib/rpc/server.rs

```diff
@@ -228,27 +233,26 @@ impl IdentityService for IdentityServer {
         code: AuthorizationCode,
     ) -> RpcResult<SessionToken> {
         // Checks if the authorization code has a valid form
-        let authorization_code = oauth::AuthorizationCode::new(code)?;
+        let authorization_code = crate::authentication::oauth::AuthorizationCode::new(code)?;
```

Comment:
Why not also import `AuthorizationCode` line 13?

Reply:
Yeah, I wanted to do that too, but then figured out the name collision between the rpc model `AuthorizationCode` and the oauth module struct. I now renamed the rpc model to `OauthAuthorizationCode`. I think this even helps to make it more clear.